CONTRIBUTING.md

```diff
@@ -35,7 +35,7 @@ There are many ways that you can contribute to the Azure SDK for Embedded C proj
 
 - For reporting bugs, requesting features, or asking for support, please file an issue in the [issues](https://github.com/Azure/azure-sdk-for-c/issues) section of the project.
 
-- If you would like to become an active contributor to this project please follow the instructions provided in [Microsoft Azure Projects Contribution Guidelines](http://azure.github.com/guidelines.html).
+- If you would like to become an active contributor to this project please follow the instructions provided in [Microsoft Azure Projects Contribution Guidelines](https://opensource.microsoft.com/collaborate).
```

Comment:
@ahsonkhan wouldn't https://opensource.guide/how-to-contribute/ be a better link replacement?